### PR TITLE
redirect

### PR DESCRIPTION
**Pull request by MaksimDannik**

### CODE REVIEW

**Review by AndrejStarusev — APPROVED** [review]

**Comment by MaksimDannik** [comment]
.

File: client/src/routing/Router.jsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { BrowserRouter, Route, Routes, Navigate} from "react-router-dom";
 import Header from "../components/header";
-import Home from "../pages/home";
+import Users from "../pages/home/Users";
```
Comment (max-baykalov) [inline client/src/routing/Router.jsx:3]: create a new folder for the new page and connect css (e.g. Users.css) file there located in the directory

File: client/src/routing/Router.jsx [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 const Router = () => (
   <BrowserRouter>
     <Header />
     <Routes>
-      <Route path="/" element={<Home />} />
+    <Route path="/users" element={<Users />} />
+    <Route path="/" element={<Navigate replace to="/users" />} />
```
Comment (max-baykalov) [inline client/src/routing/Router.jsx:10]: you could use path="/*" to handle all invalid path

File: client/src/routing/Router.jsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 import Header from "../components/header";
-import Home from "../pages/home";
+import Users from "../pages/home/Users";
 
 const Router = () => (
   <BrowserRouter>
     <Header />
     <Routes>
-      <Route path="/" element={<Home />} />
+    <Route path="/users" element={<Users />} />
```
Comment (max-baykalov) [inline client/src/routing/Router.jsx:9]: remove unused Home.jsx file and the folder

**Review by max-baykalov — CHANGES_REQUESTED** [review]

**Review by max-baykalov — APPROVED** [review]

**Pull request merged**